lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -0,0 +1,2134 @@
+const cryptoRandomString = require('crypto-random-string');
```

Comment:
data-migration2 should be removed on rds-phase-3. See this commit: https://github.com/nasa/cumulus/commit/86d4d62805dc627ec585eb6da2b341fc0c841816